pandas/tests/resample/test_period_index.py

```diff
@@ -15,8 +15,7 @@
 from pandas import DataFrame, Series, Timestamp
 from pandas.core.indexes.datetimes import date_range
 from pandas.core.indexes.period import Period, PeriodIndex, period_range
-from pandas.tests.resample.test_base import (
-    Base, resample_methods, simple_period_range_series)
+from pandas.tests.resample.test_base import Base
```

Comment:
is this needed?

Reply:
all the tests in `Base` are run for each class that inherits from `Base`. so the next pass will be to parametrize the `Base` class over the parameters in the classes that inherit from `Base` in order to be able to remove this dependency. I'll do this in a separate pass since the tests in `Base` rely on a method override so it is not trivial.